src/components/LinterMessage/index.tsx

```diff
@@ -59,22 +60,29 @@ const renderDescription = (description: LinterMessage['description']) => {
 };
 
 type PublicProps = {
+  _scrollToElement: typeof scrollToElement | null;
   inline?: boolean;
   message: LinterMessage;
 };
 
-const LinterMessageBase = ({ message, inline = false }: PublicProps) => {
+const LinterMessageBase = ({
+  _scrollToElement,
+  inline = false,
+  message,
+}: PublicProps) => {
   const { description, message: linterMessage, type } = message;
   const variant = getAlertVariant(type);
 
   return (
-    <Alert
-      className={makeClassName(styles[variant], { [styles.inline]: inline })}
-      variant={variant}
-    >
-      <Alert.Heading>{decodeHtmlEntities(linterMessage)}</Alert.Heading>
-      <p className={styles.description}>{renderDescription(description)}</p>
-    </Alert>
+    <div className="LinterMessage" ref={_scrollToElement}>
```

Comment:
btw, you can theoretically just assign the `ref` prop to `Alert` but I'm seeing what looks like a bug in the TS definitions (https://github.com/react-bootstrap/react-bootstrap/issues/3805).

Reply:
Yes, I tried that first and got the TS error you mention in that issue, which is why I created the `div`.